### PR TITLE
Ensure module logo has transparent background

### PR DESCRIPTION
**Pull request by kianmeng**
While the contributing guide forbids contributor from changing the `mix.exs` file, I believe this is an exception case.

This PR adds transparency to the module logo.

Before.
image

After.
image

After (latest).
image

### CODE REVIEW

**Comment by prodis** [comment]
@kianmeng Thanks for your PR.

The label "MISS" in the bottleneck should not be transparent.

File: mix.exs [hunk 1]
```diff
@@ -57,7 +57,7 @@ defmodule Miss.MixProject do
     [
       main: "readme",
       extras: ~w(README.md CHANGELOG.md),
-      logo: "misc/miss-elixir-logo.jpg",
+      logo: "priv/miss-elixir-logo.png",
```
Comment (prodis) [inline mix.exs]: Why did you remove the `misc` directory?

**Review by prodis — CHANGES_REQUESTED** [review]

**Comment by kianmeng** [comment]
> @kianmeng Thanks for your PR.
> 
> The label "MISS" in the bottleneck should not be transparent.

Got it.

image

File: mix.exs (hunk 1 above)
Comment (kianmeng) [reply mix.exs]: > Why did you remove the `misc` directory?

Assumed the standard `priv` directory common used for to store module asset. Let me know, I can revert back to original `misc` folder.

File: .tool-versions [hunk 2]
```diff
@@ -0,0 +1,2 @@
+erlang 21.1.4
+elixir 1.10.3-otp-21
```
Comment (prodis) [inline .tool-versions]: This file does not belong to the library. Don't add specific files for your development environment to the library.

**Review by prodis — CHANGES_REQUESTED** [review]
❓ 

https://github.com/kianmeng/miss-elixir/blob/make-logo-bg-transparent/misc/miss-elixir-logo.png

File: mix.exs (hunk 1 above)
Comment (prodis) [reply mix.exs]: Please move to `assets` and keep the original image.

File: .tool-versions (hunk 2 above)
Comment (kianmeng) [reply .tool-versions]: > This file does not belong to the library. Don't add specific files for your development environment to the library.

Fixed.

**Comment by kianmeng** [comment]
Fixed.

Original.
https://github.com/kianmeng/miss-elixir/blob/make-logo-bg-transparent/assets/miss-elixir-logo.jpg



With transparency.
https://github.com/kianmeng/miss-elixir/blob/make-logo-bg-transparent/assets/miss-elixir-logo.png

File: mix.exs (hunk 1 above)
Comment (kianmeng) [reply mix.exs]: > Please move to `assets` and keep the original image.

Done. Please verify.

**Review by prodis — APPROVED** [review]
LGTM
